hydra-node/src/Hydra/Chain/Direct/Tx/Init.hs

```diff
@@ -0,0 +1,250 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE TypeApplications #-}
+
+-- | Smart constructors for creating Hydra protocol transactions to be used in
+-- the 'Hydra.Chain.Direct' way of talking to the main-chain.
+--
+-- This module also encapsulates the transaction format used when talking to the
+-- cardano-node, which is currently different from the 'Hydra.Ledger.Cardano',
+-- thus we have not yet "reached" 'isomorphism'.
```

Comment:
Should: update/remove this module header to be accurate (I have no idea what the second paragraph is about) and either have them on all `Tx.XXX` modules or none.